react/components/Buttons/ChangeToppings.js

```diff
@@ -0,0 +1,29 @@
+import PropTypes from 'prop-types'
+import React, { Component } from 'react'
+import Button from 'vtex.styleguide/Button'
+import { injectIntl, intlShape } from 'react-intl'
+
+class ChangeToppings extends Component {
+  static propTypes = {
+    intl: intlShape.isRequired,
+    enableChangeToppings: PropTypes.bool,
+  }
+
+  render() {
+    const {
+      intl,
+      enableChangeToppings,
+    } = this.props
+    const changeLabel = intl.formatMessage({ id: 'product-customizer.change-composition' })
```

Comment:
Use `<FormattedMessage />` component from `react-intl` to internationalize messages. See the [documentation](https://github.com/yahoo/react-intl/wiki/Components#string-formatting-components) for more details.

Reply:
fixed on commit https://github.com/vtex-apps/product-customizer/commit/db328474e5ad1ce958bfe33c8d362918a7c56924

Reply:
@henriquecaraujo you don't need to comment on these issues, github already notifies us.